osu.Game/Graphics/UserInterface/BackButton.cs

```diff
@@ -0,0 +1,145 @@
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                            Colour = new Color4(195, 40, 140, 255),
+                            Shear = new Vector2(0.1f, 0),
+                        },
+                        icon = new TextAwesome
+                        {
+                            Anchor = Anchor.Centre,
+                            TextSize = 25,
+                            Icon = FontAwesome.fa_osu_left_o
+                        },
+                    }
+                },
+                rightContainer = new Container
+                {
+                    Origin = Anchor.TopLeft,
+                    Anchor = Anchor.TopLeft,
+                    RelativeSizeAxes = Axes.Both,
+                    Position = Position + InitialExtendLength,
+                    Children = new Drawable[]
+                    {
+                        textBox = new Box
+                        {
+                            Colour = new Color4(238, 51, 153, 255),
+                            Origin = Anchor.TopLeft,
+                            Anchor = Anchor.TopLeft,
+                            RelativeSizeAxes = Axes.Both,
+                            Shear = new Vector2(0.1f, 0), 
+                            EdgeSmoothness = new Vector2(1.5f, 0), 
+                        },
+                        new SpriteText
+                        {
+                            Origin = Anchor.Centre,
+                            Anchor = Anchor.Centre,
+                            Text = @"Back",
+                        }
+                    }
+                }
+            };
+        }
+        protected override bool OnHover(InputState state)
+        {
+            icon.ClearTransformations();
+            rightContainer.ClearTransformations();
+            leftContainer.ClearTransformations();
+
+            rightContainer.MoveTo(Position + ExtendLength, transformTime, EasingTypes.OutElastic);
+            leftContainer.ResizeTo(new Vector2(ExtendLength.X, 1.0f), transformTime, EasingTypes.OutElastic);
+
+            int duration = 0; //(int)(Game.Audio.BeatLength / 2);
+            if (duration == 0) duration = 250;
+
+            double offset = 0; //(1 - Game.Audio.SyncBeatProgress) * duration;
+            double startTime = Time.Current + offset;
+
+            // basic pulse
+            icon.Transforms.Add(new TransformScale
+            {
+                StartValue = new Vector2(1.1f, 1.1f),
+                EndValue = Vector2.One,
+                StartTime = startTime,
+                EndTime = startTime + duration,
+                Easing = EasingTypes.Out,
+                LoopCount = -1,
+                LoopDelay = duration
+            });
+
+            return true;
+        }
+
+        protected override void OnHoverLost(InputState state)
+        {
+            icon.ClearTransformations();
+            rightContainer.ClearTransformations();

```

Comment:
the ClearTransformations are unnecessary on left/right containers since you are overwriting them below.

the icon is probably necessary to remove the pulse (although in stable the pulse was still present when not hovered, just with less velocity. you may want to implement this if you're going for 1:1 behaviour).